src/redist/targets/packaging/windows/clisdk/bundle.thm

```diff
@@ -13,16 +13,16 @@
     <Page Name="Help">
         <Text X="0" Y="0" Width="660" Height="75" FontId="1" />
 
-        <Text X="11" Y="80" Width="-11" Height="30" FontId="2" DisablePrefix="yes">#(loc.HelpHeader)</Text>
-        <Text X="20" Y="115" Width="-11" Height="-35" FontId="3" DisablePrefix="yes">#(loc.HelpText)</Text>
+        <Text Name="HelpHeader" X="11" Y="80" Width="-11" Height="30" FontId="2" DisablePrefix="yes">#(loc.HelpHeader)</Text>
+        <Text Name="HelpText" X="20" Y="115" Width="-11" Height="-35" FontId="3" DisablePrefix="yes">#(loc.HelpText)</Text>
         <Button Name="HelpCancelButton" X="-11" Y="-11" Width="75" Height="23" TabStop="yes" FontId="0">#(loc.HelpCloseButton)</Button>
-        <Image X="0" Y="0" Width="620" Height="418" ImageFile="bg.png" />
+        <Image Name="BackGroundImage" X="0" Y="0" Width="620" Height="418" ImageFile="bg.png" />
```

Comment:
nit: Since background is one word, I would suggest "BackgroundImage"

Reply:
Sure...